pixeltable/env.py

```diff
@@ -136,6 +141,16 @@ def set_up(self, echo: bool = False) -> None:
         self._log_dir = self._home / 'logs'
         self._tmp_dir = self._home / 'tmp'
         self._pgdata_dir = Path(os.environ.get('PIXELTABLE_PGDATA', str(self._home / 'pgdata')))
+        self._config_file = self._home / 'pixeltable.yaml'
```

Comment:
config.yaml?

precede w/ blank line?

also, do we need to keep this or is it only used once?

Reply:
I kind of want to keep it. It's useful for debugging, and it's a lightweight structure. But I don't feel super strongly about it